### PR TITLE
Core/tests

### PR DESCRIPTION
**Pull request by jgfidelis**
just add tests to product wrapper

### CODE REVIEW

File: react/__tests__/ProductWrapper.test.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+      product: newProduct,
+      params: { slug: newProduct.linkText },
+      productQuery: { product: newProduct, loading: false },
+    }
+    rerender(
+      <ProductWrapper {...getProps(newProps)}>
+        <ProductPageMock />
+      </ProductWrapper>
+    )
+    act(() => {})
```
Comment (lbebber) [inline react/__tests__/ProductWrapper.test.js]: perhaps add a short comment on why this is needed

File: react/__tests__/ProductWrapper.test.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+      ...props,
+      query: { skuId: itemtwo.itemId },
+    }
+    rerender(
+      <ProductWrapper {...getProps(newProps)}>
+        <ProductPageMock />
+      </ProductWrapper>
+    )
+
+    act(() => {})
```
Comment (lbebber) [inline react/__tests__/ProductWrapper.test.js]: ditto

File: react/__tests__/ProductWrapper.test.js [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+      </ProductWrapper>
+    )
+  }
+
+  it('should render with correct title and no meta', () => {
+    const product = getProduct()
+    const { queryByTestId, getByText } = renderComponent()
+    getByText(new RegExp(product.titleTag))
+    // Doesnt have metatag description, <meta> should not be rendered
+    expect(queryByTestId('descritpion')).toBeNull()
```
Comment (klzns) [inline react/__tests__/ProductWrapper.test.js]:
```suggestion
    expect(queryByTestId('description')).toBeNull()
```

File: react/__tests__/ProductWrapper.test.js [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+    const newProduct = getProduct({
+      items: [newItem],
+      productId: '2',
+      linkText: 'product-slug-2',
+      titleTag: 'Product 2',
+      productName: 'Product 2',
+    })
+
+    getByText(new RegExp(product.titleTag))
+    getByText(`Selected Item id: ${product.items[0].itemId}`)
```
Comment (klzns) [inline react/__tests__/ProductWrapper.test.js]: Create helper functions to check those info.

Exameple:

```jsx
const { getItemIdm, getItemName } = renderComponent()

getItemId(product.items[0].itemId)
getItemName(product.items[0].name)
```

**Pull request merged**